code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubClient.java

```diff
@@ -104,16 +106,42 @@ public GithubAccountInfo getUserAccountInfo(String token) throws IOException {
         return orgInfoList;
     }
 
-    public GithubAccountInfo getUserIdByName(String username) throws IOException {
+    public List<TeamAccountInfo> getOrgTeamInfo(String token, String org) throws IOException {
+        if (StringUtils.isEmpty(token)) {
+            return null;
+        }
+        List<TeamAccountInfo> teamInfoList = new ArrayList<>();
+        List<Map<String, Object>> jsonData = null;
+
+        HttpResponse response = Request.Get(GITHUB_UNAUTH_ORG_URL.get() + org + "/teams").addHeader("Authorization", "token " + token)
```

Comment:
This URL does not require auth info. But is rate limited to 60/hr. So, by adding auth info to unauth url, we get a higher Rate-Limit. The Variable itself represent that URL that need not be authorized